src/Stepper/Step.jsx

```diff
@@ -0,0 +1,340 @@
+import React, {PropTypes} from 'react';
+
+import {
+  TouchRipple,
+} from '../ripples';
+
+import Avatar from '../avatar';
+import Mixins from '../mixins';
+
+import {getMuiTheme} from '../styles';
+import ReactDOM from 'react-dom';
+
+const Step = React.createClass({
+  propTypes: {
+    children: PropTypes.node,
+
+    /*
+      Customize the style of children wrapper div (include control button group and children)
+    */
+    childrenWrapperStyle: PropTypes.object,
+    /*
+      Customize the style of connector line
+    */
+    connectorLineStyle: PropTypes.object,
+     /*
+      Should be control buttons to handle how to move to next step
+    */
+    controlButtonsGroup: PropTypes.node,
+     /*
+     Customize the style of control button group wrapper
+    */
+    controlButtonsGroupWrapperStyle: PropTypes.object,
+     /*
+     the step is selected or not
+    */
+    isActive: PropTypes.bool,
+     /*
+     the step is completed or not
+    */
+    isCompleted: PropTypes.bool,
+     /*
+     the step is last step or not (which passed by automatically from Steps wrapper parent, no need
+     to pass directly)
+    */
+    isLastStep: PropTypes.bool,
```

Comment:
@alitaheri Are props like this what we're planning to use the `@internal` tag for? Or `@ignore`?